dags/app_store_analytics.py

```diff
@@ -0,0 +1,67 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+from utils.gcp import gke_command
+
+
+default_args = {
+    "owner": "bewu@mozilla.com",
+    "depends_on_past": False,
+    "start_date": datetime(2020, 6, 23),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=30),
+    "email": [
+        "bewu@mozilla.com",
+    ],
+}
+
+PROJECT_ID = "moz-fx-data-marketing-prod"
+DATASET_ID = "apple_app_store_export"
+
+apps = [
+    ("989804926", "Firefox"),
+    ("1489407738", "VPN"),
+    ("1295998056", "WebXRViewer"),
+    ("1314000270", "Lockwise"),
+    ("1073435754", "Klar"),
+    ("1055677337", "Focus"),
+]
+
+with DAG("app_store_analytics",
+         default_args=default_args,
+         max_active_runs=1,
+         schedule_interval="@daily") as dag:
+
+    tasks = []
+
+    # App exports are scheduled sequentially to avoid hit api rate limit
+    for i, (app_id, app_name) in enumerate(apps):
+        commands = [
+            "yarn",
+            "--silent",  # silent to hide arguments from logs
+            "export",
+            "--username={{ var.value.app_store_connect_username }}",
+            "--password={{ var.value.app_store_connect_password }}",
+            f"--app-id={app_id}",
+            f"--app-name={app_name}",
+            "--start-date={{ yesterday_ds }}",  # previous day data is incomplete
+            f"--project={PROJECT_ID}",
+            f"--dataset={DATASET_ID}",
+        ]
+
+        if i == 0:
+            commands.append("--overwrite")
```

Comment:
What's the purpose of this conditional? Can you add a comment?

Reply:
This clears the current day's partition in the destination tables to ensure the only data for the day is the data written by this dag run.  This means if the table has any apps that aren't recognized here they'll be deleted.